frontend/src/views/CellCards/components/CellCard/components/Description/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { CellCardDescription, Source, Wrapper } from "./style";
+import { useDescription, useClDescription } from "src/common/queries/cellCards";
+import { Tooltip } from "czifui";
+import Link from "../common/Link";
+import { StyledLink } from "../common/Link/style";
+
+export const CELL_CARD_CL_DESCRIPTION = "cell-card-cl-description";
+export const CELL_CARD_GPT_DESCRIPTION = "cell-card-gpt-description";
+export const CELL_CARD_GPT_TOOLTIP_LINK = "cell-card-gpt-tooltip-link";
+
+interface DescriptionProps {
+  cellTypeName: string;
+}
+export default function Description({
+  cellTypeName,
+}: DescriptionProps): JSX.Element {
+  const [descriptionGpt, setDescriptionGpt] = useState<string>("");
+  const [descriptionCl, setDescriptionCl] = useState<string>("");
+
+  const router = useRouter();
+  const { cellTypeId: cellTypeIdRaw } = router.query;
+  const cellTypeId = (cellTypeIdRaw as string)?.replace("_", ":") ?? "";
```

Comment:
Why do this in both parent and child instead of passing as a prop from the parent and keeping this component a little more dumb?